staging/src/k8s.io/apiserver/pkg/storage/etcd3/lease_manager.go

```diff
@@ -22,8 +22,28 @@ import (
 	"time"
 
 	"go.etcd.io/etcd/clientv3"
+	"k8s.io/apiserver/pkg/storage/etcd3/metrics"
 )
 
+const (
+	DefaultLeaseReuseDurationSeconds = 60
+	DefaultLeaseMaxObjectSize        = 1000000
+)
+
+type LeaseManagerConfig struct {
```

Comment:
I like this.
You may want to extract the introduction of this LeaseManagerConfig to a separate PR (with just a single ReuseDurationSeconds field) as this is clear improvement and we can have that already merged.

Reply:
:thinking: I think in previous PR we decide to make a LeaseManagerConfig at this PR https://github.com/kubernetes/kubernetes/pull/97009#discussion_r535950842

Reply:
I didn't know if this PR will happen or not back then - now I see it :)
But if you want to leave in this PR - that's fine too.